pages/api/avo/index.ts

```diff
@@ -17,6 +17,8 @@ const allAvos = async (req: IncomingMessage, res: ServerResponse) => {
     // https://nextjs.org/docs/api-routes/response-helpers
     res.statusCode = 200
     res.setHeader('Content-Type', 'application/json')
+    res.setHeader('Access-Control-Allow-Origin', '*')
```

Comment:
This method was already added on `enablePublicAccess` but it doesn't hurt making it more explicit.

Reply:
I thought it. But it was weird because I was trying a GET request to `https://platzi-avo.vercel.app/api/avo/` and it didn't work. I was getting the error `Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://platzi-avo.vercel.app/api/avo/. (Reason: CORS header ‘Access-Control-Allow-Origin’ missing).`

Reply:
I see. And is it fixed now?

Reply:
Yes it is!!